project/Dependencies.scala

```diff
@@ -79,6 +79,7 @@ object Dependencies {
     ExclusionRule("org.apache.ivy", "ivy"),
     ExclusionRule("javax.servlet", "servlet-api"),
     ExclusionRule("org.mortbay.jetty", "servlet-api"),
+    // P.S. Need to comment this out for spark 1.5.2 work with Hive+parquet in CDH 5.4.4+ (hadoop 2.6.0)    
     ExclusionRule("com.twitter", "parquet-hadoop-bundle")
```

Comment:
a fancy if might be better, but don't want to tinker with this. commenting it out makes it work for us.

the issue seem to be the fact that `parquet-format & parquet` moved their namespaces from `parquet.*` into `org.apache.parquet` recently. spark 1.5, uses the new parquet.

hive libs (likely the ones bundled into `spark-notebook` ? ) on the over hand, seem to use the old lib-path.
I don't think that we include any CDH hive libs (except for hadoop natives).